docs/openfaas-pro/jetstream.md

```diff
@@ -18,25 +19,168 @@ On our blog we demo and explore some architectural patterns for these uses cases
 - [Generate PDFs at scale on Kubernetes using OpenFaaS and Puppeteer](https://www.openfaas.com/blog/pdf-generation-at-scale-on-kubernetes/)
 - [How to process your data the resilient way with back pressure](https://www.openfaas.com/blog/limits-and-backpressure/)
 
+## Terminology
+
+In JetStream ("js" for short), there are new terms that will help us all in running and debugging the product.
+
+1. A JetStream Server is the original NATS Core project, running in "jetstream mode"
+2. A Stream is a set of messages, and works in a similar way to Kafka
+3. A Consumer is a group that various Subscribers can join to read messages and keep track of where they left off
+5. A Subscriber is what the queue-worker creates, if the max_inflight is set to 25, the queue-worker will create 25 subscribers
+
+> You can learn more about JetStream here: [Docs for JetStream](https://docs.nats.io/nats-concepts/jetstream)
+
+## Installation
+
+For staging and development environments OpenFaaS can be deployed with an embedded version of the NATS server which uses an in-memory store.
+
+To enable JetSteam for OpenFaaS set `jetstream` as the queue mode in the values.yaml file of the [OpenFaaS Helm chart](https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/README.md)
+
+```yaml
+queueMode: jetstream
+nats:
+    streamReplication: 1
+```
+
+If the NATS pod restarts, you will lose all messages that it contains. In your development or staging environment, this shouldn't happen very often.
+
+For production environments you will need to install NATS separately using its Helm chart with at least 3 server replicas, so that if a pod crashes, the data can be recovered automatically.
+
+```yaml
+queueMode: jetstream
+nats:
+  streamReplication: 3
+  external:
+    enabled: true
+    host: "nats.nats"
+    port: "4222"
+```
+
+## Configure JetStream
```

Comment:
This advanced section belongs at the bottom of the file, since we want this to be like a brochure for business people, or developers who are browsing.

Reply:
I moved this section to the bottom.